tests/puppeteer/pages/BO/orders/invoices/index.js

```diff
@@ -29,6 +29,9 @@ module.exports = class Invoice extends BOBasePage {
     this.taxBreakdownEnable = `${this.invoiceOptionsForm} label[for="form_invoice_options_enable_tax_breakdown_%ID"]`;
     this.invoiceOptionEnableProductImage = `${this.invoiceOptionsForm} 
     label[for="form_invoice_options_enable_product_images_%ID"]`;
+    this.invoiceNumberInput = `${this.invoiceOptionsForm} #form_invoice_options_invoice_number`;
+    this.legalFreeTextInput = `${this.invoiceOptionsForm} #form_invoice_options_legal_free_text_1`;
```

Comment:
No need to add a container when it's an id. If there is more than one id, it's a bug in the Core and must be fixed :D